src/main/java/net/snowflake/ingest/streaming/internal/ChannelCache.java

```diff
@@ -14,24 +14,25 @@
  * during flush. The key is a fully qualified table name and the value is a set of channels that
  * belongs to this table
  */
-class ChannelCache {
+class ChannelCache<T> {
```

Comment:
This might just be me but I am not in favor of Single Letter generics. (I know this is the java norm and is standard but it would be much more readable if it is not T and something meaningful even though mouthful)

Or may be a single line description of what T is?

Reply:
sure, I forgot about the comment here!